schedule/containers/sle_image_on_sle_host.yaml

```diff
@@ -12,6 +12,17 @@ schedule:
   - containers/container_diff
   - '{{validate_btrfs}}'
 conditional_schedule:
+  boot:
+    ARCH:
+      's390x':
+        - installation/bootloader_start
+        - boot/boot_to_desktop
```

Comment:
you are already scheduling this in line 7, why repeating it?

Reply:
7f7d1aa26